app/lib/katello/resources/candlepin.rb

```diff
@@ -399,6 +399,10 @@ def import(organization_name, path_to_file, options)
             self.post(path, {:import => File.new(path_to_file, 'rb')}, self.default_headers.except('content-type'))
           end
 
+          def product_content(organization_name)
+            Product.all(organization_name, [:id, :productContent])
```

Comment:
This is no chance this would be confused with '::Katello::Product' right ?

Reply:
I don't think so, we do things in similar places in this file